tests/test_kernel32/test_handle.py

```diff
@@ -120,3 +122,95 @@ def test_process_dies_before_timeout(self):
         self.assertEqual(
             WaitForSingleObject(hProcess, library.INFINITE),
             library.WAIT_OBJECT_0)
+
+
+class TestGetHandleInformation(TestCase):
+    """
+    Tests for :func:`pywincffi.kernel32.GetHandleInformation`
+    """
+    def _expected_inheritance(self):
+        # Python >= 3.4 creates non-inheritable handles (PEP 0446)
+        if sys.hexversion >= 0x30400f0:
+            return 0
+        else:
+            return 1
+
+    def test_get_handle_info_stdin(self):
+        _, library = dist.load()
+        handle_stdin = GetStdHandle(library.STD_INPUT_HANDLE)
+        inherit = GetHandleInformation(handle_stdin) & 1
+        expected = self._expected_inheritance()
+        self.assertEqual(inherit, expected)
+
+    def test_get_handle_info_file(self):
+        # can't use mkstemp: not inheritable on Python < 3.4
+        dirname = tempfile.mkdtemp()
+        filename = os.path.join(dirname, "test_file")
+        test_file = open(filename, "w")
+        test_file.write("data")
+        file_handle = handle_from_file(test_file)
```

Comment:
Since you're not using a `with open(...)` I suggest you use `try: finally:` to ensure `close()` is called so the cleanup functions shouldn't fail if parts of the test fails:

```python
try:
    # everything after the open and before close()
finally:
    test_file.close()
```